src/commands/deploy.js

```diff
@@ -212,7 +222,6 @@ const runDeploy = async ({
   const logsUrl = `${get(results, 'deploy.admin_url')}/deploys/${get(results, 'deploy.id')}`
 
   return {
-    name: results.deploy.deployId,
```

Comment:
`results.deploy.deployId` is always `undefined`, this should be either `results.deploy.id` or `results.deployId`.
Since `name` was never printed, and `deployId` is already part of the result, I chose to remove the line entirely.